<launch>
	<!-- algorithm param -->
	<arg name="sensor_range" default="30.0" />
	<arg name="octomap_resolution" default="0.3" />
	<arg name="octomap_hit" default="0.8" />
	<arg name="octomap_miss" default="0.4" />
	<arg name="octomap_hz" default="10.0" />
	<arg name="robot_size" default="0.5" />
	<arg name="sensor_reliable_range" default="20.0" />
	<arg name="sensor_angle" default="0.523599" /><!-- 0.523599-->>
	<arg name="update_tolerance" default="0.02" />
	
	<!-- map bound -->
	<arg name="map_min_x" default="-58.0" /><!---5.0 -->
	<arg name="map_min_y" default="-17.0" /><!---5.0 -->
	<arg name="map_min_z" default= "0.0" />  <!--1.0" -->
	<arg name="map_max_x" default="32.0"  /> <!--65.0-->
	<arg name="map_max_y" default="53.0"  /> <!--20.0-->
	<arg name="map_max_z" default="20.0"  /> <!--25.0-->
	
	<param name="sensor_range" value="$(arg sensor_range)" />
	<param name="octomap_resolution" value="$(arg octomap_resolution)" />
	<param name="octomap_hit" value="$(arg octomap_hit)" />
	<param name="octomap_miss" value="$(arg octomap_miss)" />
	<param name="octomap_hz" value="$(arg octomap_hz)" />
	<param name="robot_size" value="$(arg robot_size)" />
	<param name="sensor_reliable_range" value="$(arg sensor_reliable_range)" />
	<param name="sensor_angle" value="$(arg sensor_angle)" />
	<param name="update_tolerance" value="$(arg update_tolerance)" />
	<param name="map_min_x" value="$(arg map_min_x)" />
	<param name="map_min_y" value="$(arg map_min_y)" />
	<param name="map_min_z" value="$(arg map_min_z)" />
	<param name="map_max_x" value="$(arg map_max_x)" />
	<param name="map_max_y" value="$(arg map_max_y)" />
	<param name="map_max_z" value="$(arg map_max_z)" />

	<node name="main" pkg="srsd_planner" type="main" output="screen"/>
	<node pkg="tf2_ros" type="static_transform_publisher" name="velodyne_link_broadcaster" args="0.08 0 0.01 0 0 0 1 base_link velodyne" />
</launch>

